packages/dds/tree/src/test/shared-tree/summary.bench.ts

```diff
@@ -51,20 +51,36 @@ describe("Summary benchmarks", () => {
 		});
 		for (const [numberOfNodes, minLength, maxLength] of nodesCountWide) {
 			it(`a wide tree with ${numberOfNodes} nodes.`, async () => {
-				const summaryTree = getSummaryTree(makeWideContentWithEndValue(numberOfNodes, 1));
-				const summaryString = JSON.stringify(summaryTree);
-				const summarySize = IsoBuffer.from(summaryString).byteLength;
-				assert(summarySize > minLength);
-				assert(summarySize < maxLength);
+				benchmarkCustom({
+					only: false,
+					type: BenchmarkType.Measurement,
+					title: `a wide tree with ${numberOfNodes} nodes.`,
+					run: async (reporter) => {
+						const summaryTree = getSummaryTree(makeWideContentWithEndValue(numberOfNodes, 1));
+						const summaryString = JSON.stringify(summaryTree);
+						const summarySize = IsoBuffer.from(summaryString).byteLength;
+						reporter.addMeasurement("summarySize", summarySize);
+						assert(summarySize > minLength);
+						assert(summarySize < maxLength);
```

Comment:
Extract this to a helper function and reuse it below as well

Reply:
I also thought about deduping the logic, however, decided not to do so since I am not too familiar with the usage of functions in [scalableTestTree.ts](https://github.com/microsoft/FluidFramework/blob/main/packages/dds/tree/src/test/scalableTestTrees.ts).

For example, `getSummaryTree()` accepts `TreeContent` type (which is a return type of `makeDeepContent()` and `makeWideContentWithEndValue()`). However, there are also other helper functions within the file with the naming of `make--Foo--()` with different return types. 

I think the benefit of creating a helper function will be greater if there exists more `make--Foo--()` functions with return type of `TreeContent`. However, for now, considering the usage of `benchmarkCustom()`, dedup isn't super necessary.

Interested to have your thought.

Reply:
I think it's fine to leave test case this way since we are creating deep/wide tree and then verify their summary size inside benchmark run function. Showing the test steps with tree's structure(deep/wide) looks clearer to me.

Reply:
Dedupe this part except the creation of `summaryTree`.

Reply:
@chentong7 I hope I met the right compromise of concise yet readable logic. Interested in your thoughts.